tests/testscript.py

```diff
@@ -0,0 +1,22 @@
+import pytng
+
+file_iterator = pytng.TNGFileIterator("./reference_files/water_uncompressed_vels_forces.tng", mode="r", debug=False)
+file_iterator.read_frame(0)
+file_iterator.read_frame(10000) # this integrator timestep has data
```

Comment:
See here for how the interface looks currently.

Reply:
is this the 10,000th frame in the file, or the 10,000th of the simulation?

Reply:
10,000th of the simulation, the frame interface is one of the things I still think needs a fair bit of improvement.

One idea is possibly to iterate over the greatest common divisor of the data strides?